Makefile

```diff
@@ -29,7 +29,7 @@ verilator      ?= verilator
 # traget option
 target-options ?=
 # additional definess
-defines        ?= WT_DCACHE+RVFI_TRACE
```

Comment:
Unfortunately removing RFVI_TRACE will make all the Thales CI fail

Reply:
Why? It looks to me like `RVFI_TRACE` is defined via the `isscomp_opts` variables when it's actually needed:
https://github.com/openhwgroup/core-v-verif/blob/521dbba95a408adb95965451dceec696e877f094/cva6/sim/cva6.py#L757-L758
I feel like the common users of this script do not want `RVFI_TRACE`, especially when executing software that contains `ecall` instructions such as the riscv-tests. I don't think that terminating on an `ecall` should be the default behaviour of the CVA6 simulation. It took us quite some git-bisecting to find out why the riscv-tests weren't passing anymore, and the same might happen to others.

Reply:
You are right. The ultimate solution is to pass RVFI_TRACE through the isscomp_opts option. As it is not the case Today, all teh Thales CI will fail. If I gettime, I will try to fix it.

Reply:
Today we change our mind. We will modify the end of test to stop on HTIF condition (write a special data in to_host address) and no more on ecall. I let you know.